DigitalLearningSolutions.Web/Controllers/ResetPasswordController.cs

```diff
@@ -0,0 +1,39 @@
+namespace DigitalLearningSolutions.Web.Controllers
+{
+    using System.Threading.Tasks;
+    using DigitalLearningSolutions.Data.Services;
+    using Microsoft.AspNetCore.Mvc;
+
+    public class ResetPasswordController : Controller
+    {
+        private readonly IPasswordResetService passwordResetService;
+        public ResetPasswordController(IPasswordResetService passwordResetService)
+        {
+            this.passwordResetService = passwordResetService;
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> Index(string email, string resetHash)

```

Comment:
The query parameter "resetHash" here does not match the query parameter name in the email generated. When the email is created, it uses the parameter name "code" in the link instead. So when clicking the link in the reset password email, it won't be able to find "resetHash" and instead that will come up null. I think you can either change "resetHash" to "code" here, or you can change the link generation to use "resetHash" instead and keep this the same.

I tested renaming the "resetHash" to "code" and that allowed the reset password hash in the email link to be picked up.